supereeg/brain.py

```diff
@@ -342,7 +357,7 @@ def get_slice(self, sample_inds=None, loc_inds=None, inplace=False):
         date_created = time.strftime("%c")
 
         b = Brain(data=data, locs=locs, sessions=sessions, sample_rate=sample_rate, meta=meta, date_created=date_created,
-                  filter=copy.copy(self.filter), kurtosis=kurtosis)
+                  filter=None, kurtosis=kurtosis)
         if inplace:
             self = b
```

Comment:
does this work?  i would have thought you'd need something like `self.__init__(b)`